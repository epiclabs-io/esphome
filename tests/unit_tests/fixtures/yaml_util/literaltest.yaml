---
substitutions:
  pins: [14, 15, 33, 39]

esphome:
  name: test

binary_sensor:
  - platform: gpio
    pin: 19
    name: !literal 'some ${pin} name' # ${pin} will not be expanded because of !literal tag
