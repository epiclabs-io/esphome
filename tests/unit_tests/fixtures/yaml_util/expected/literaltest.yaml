binary_sensor:
- name: some ${pin} name
  pin: 19
  platform: gpio
esphome:
  name: test
